examples/silly_clock.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python2
```

Comment:
this should not be tied to python2.. or is this example py2 only?

Reply:
Agreed - nothing py2-specific here.

Reply:
@ttsiodras can you change this in a new PR?

Reply:
> @ttsiodras can you change this in a new PR?

Done - opened [a new one](https://github.com/rm-hull/luma.led_matrix/pull/184).